test/bugs/ui-4071.js

```diff
@@ -35,7 +35,8 @@ const spec = {
               },
               encoding: {
                 industries: {
-                  style: 'form'
+                  style: 'form',
+                  explode: false
```

Comment:
If style is `form`, the default value of explode is `true`. However, this test seemed to assume the behavior when explode was `false`, so I set explode to `false`.